src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4428,8 +4428,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
  /optimize[+|-]                Enable optimizations (Short form: /o)
  /deterministic                Produce a deterministic assembly
                                (including module version GUID and timestamp)
- /instrument:options           Sets instrumentation options.
-                               If empty instrumentation is disabled.
+ /instrument:{TestCoverage}    Specify instrumentation method.
+                               If flag is not passed then instrumentation is disabled.
```

Comment:
I'm not sure precisely what this means. Is "flag" "instrument" or "TestCoverage"?

I don't know if there is a way to word this such that adding a new kind of instrumentation involves only adding text.